example/src/main/java/com/stripe/example/activity/CreateCardPaymentMethodActivity.kt

```diff
@@ -129,4 +139,8 @@ class CreateCardPaymentMethodActivity : AppCompatActivity() {
             }
         }
     }
+
+    private companion object {
+        private const val TAG = "CardValidCallback"
```

Comment:
The value of this tag seems unexpected?

Reply:
was trying to use `String` "CreateCardPaymentMethodActivity" for the tag, but it's too long to be a loggcat tag. Renamed to a more verbose name for card valid callback